packages/arb-token-bridge-ui/src/hooks/TransferPanel/useGasSummary.ts

```diff
@@ -159,59 +237,32 @@ export function useGasSummary(): UseGasSummaryResult {
     isOft
   ])
 
-  const gasSummary: UseGasSummaryResult = useMemo(() => {
-    if (
-      !isDepositMode &&
-      (isTokenArbitrumOneNativeUSDC(selectedToken?.address) ||
-        isTokenArbitrumSepoliaNativeUSDC(selectedToken?.address))
-    ) {
-      return {
-        status: 'unavailable',
-        estimatedParentChainGasFees: undefined,
```

Comment:
This no longer specifies `undefined` for this case, just returning `estimatedParentChainGasFees` as it was.

I checked the UI for native USDC withdrawal from Arb Sepolia to Sepolia and it looks fine